bosh-upload-release/task.yml

```diff
@@ -0,0 +1,28 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: relintdockerhubpushbot/cf-deployment-concourse-tasks
+    tag: v7.5.0
+
+inputs:
+- name: cf-deployment-concourse-tasks  # - This repo
+- name: bbl-state  # - The repo containing the Director's bbl state dir
+- name: tarballs  # - Directory containing the tarball to upload to the director
+- name: version  # - Directory containing the version of the tarball to upload to the director
+
+run:
+  path: cf-deployment-concourse-tasks/bosh-upload-release/task
+
+params:
+  BBL_STATE_DIR: bbl-state
+  # - Path to the directory containing the `bbl-state.json` file
+  # - The path is relative to the `bbl-state` input
+  # - If blank or '.', uses `bbl-state` input as the location for bbl state
+  # - If the specified path does not exist, it will be created
+
+  TARBALL_NAME: release
```

Comment:
Two thoughts. Does it make sense for this parameter to have a default value? As far as we can see, the pipeline will always have to specify which release should be uploaded, so the default will never be used. Also, maybe `RELEASE_NAME` would make more sense?

Reply:
I'll remove the default value. I called it `TARBALL_NAME` because in the case of UAA, the tarball prefix is `uaa-release-<version>.tgz` whereas our release name is `uaa`. Is that a blocker?

Reply:
Maybe `TARBALL_PREFIX` then?